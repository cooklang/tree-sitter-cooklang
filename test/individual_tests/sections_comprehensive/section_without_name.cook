===

This is an unnamed section.