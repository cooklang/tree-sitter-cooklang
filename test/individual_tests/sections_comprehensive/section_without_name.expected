(recipe
  (section
    header: (section_header)
    content: (step
      (plain_text))))